packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -118,6 +118,13 @@ export interface QueueProcessingServiceBaseProps {
    * @default false
    */
   readonly enableECSManagedTags?: boolean;
+
+  /**
+   * The name of a family that the task definition is registered to. A family groups multiple versions of a task definition.
+   *
+   * @default - Automatically generated name.
```

Comment:
The default should be none as you're not actually generating a family name.

Reply:
Good catch. Had copied this verbatim from `CommonTaskDefinitionProps` for consistency.

Reply:
Hold on. `task-defnition.js` actually auto-generates it. Will add this to existing unit tests to make it clearer.

Reply:
The reason I was leaning towards none is because this construct does not auto-generate the name itself. If the task definition family name generation logic is changed in the future, we'll need to update the comment here as well.

Reply:
Agree. Added it in unit test for that eventuality. It should break if the behaviour changes.